labels.json

```diff
@@ -5071,127 +5396,12 @@
                 "lang": "en",
                 "name": "Ender Dragon",
                 "description": "This user is an Ender Dragon!"
-            }
```

Comment:
Hello, mind to re-add the removed content? Looks like this commit removes some labels. Thanks!

Reply:
Did you add more labels? Because when I opened for the first time, it ended on Ender Dragon.

![photo_2024-08-31_01-57-13](https://github.com/user-attachments/assets/79322529-6ada-47e2-a468-797ca89571fa)

Reply:
Yep there are more labels :) You might have to pull in the latest changes from upstream (my copy of the repo) into yours, handle the merge, and then push that back to this branch.

Reply:
Sorry, I'm not used to GitHub, so I don't really know how to do that. 😣
Can you just cancel my commit, then I add the new labels and commit again?

Reply:
You're welcome to do it that way sure :)
Here's some potentially helpful reading for pulling in the upstream changes, if you do want to try that out: https://stackoverflow.com/a/52981214

But also you can make a new branch and new PR if you'd like